store/modules/events.ts

```diff
@@ -33,8 +42,13 @@ export default {
     }
   },
   mutations: {
-    setEvents (state, events) {
-      state.communityEvents = events
+    setEvents (state, payload) {
+      const { events, eventsList } = payload
+
+      state[events] = eventsList
+    },
+    setActiveTab (state, payload) {
+      state.activeTab = payload
```

Comment:
If you're not destructuring the payload cause it is a simple type, it'd be better to use a more meaningful name.

Reply:
I think `setActiveTab` and `state.activeTab` give enough meaningful name. I like the use of `payload` and how it follows a consistency with the rest of mutation methods